tests/integration/test_s3.py

```diff
@@ -696,6 +697,49 @@ def test_s3_event_notification_with_sqs(self):
         self.sqs_client.delete_queue(QueueUrl=queue_url)
         self._delete_bucket(TEST_BUCKET_WITH_NOTIFICATION, [key_by_path])
 
+    def test_s3_delete_object_with_version_id(self):
+        test_1st_key = 'aws/s3/testkey1.txt'
+        test_2nd_key = 'aws/s3/testkey2.txt'
+
+        # flake8: noqa: W291
+        body = """ Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. 
```

Comment:
Looks like we have already defined the same string in this file. I guess in the future we can define it as a shared constant, or simply generate a string on the fly, like:
```
body = 'Lorem ipsum dolor sit amet ... ' * 20
```